pkg/project/api/validation/validation.go

```diff
@@ -12,12 +11,14 @@ import (
 )
 
 // ValidateProject tests required fields for a Project.
-func ValidateProject(project *api.Project) fielderrors.ValidationErrorList {
+func ValidateProject(project *api.Project, create bool) fielderrors.ValidationErrorList {
```

Comment:
This was calling `ValidateProject()`, and I couldn't tell how it was used: https://github.com/openshift/origin/pull/2351/files#diff-f19a7777a36882029db668a427f950feL71